code/iaas/config-item/server/src/main/java/io/cattle/platform/configitem/context/dao/impl/MetaDataInfoDaoImpl.java

```diff
@@ -219,7 +219,7 @@ protected HostMetaData map(List<Object> input) {
             }
         };
 
-        HostTable host = mapper.add(HOST);
+        HostTable host = mapper.add(HOST, HOST.STATE);
```

Comment:
My understanding is that add(HOST) would add all the fields from host, including state. Is that not the case?

Reply:
I guess you are right, will remove this line and test.